source/includes/write/insert.kt

```diff
@@ -0,0 +1,42 @@
+import com.mongodb.client.model.DeleteOptions
+import com.mongodb.client.model.Filters.*
+import com.mongodb.client.model.InsertManyOptions
+import com.mongodb.kotlin.client.MongoClient
+
+// start-data-class
+data class Restaurant(val name: String, val borough: String)
+// end-data-class
+
+fun main() {
+    val uri = "<connection string>"
+
+    val mongoClient = MongoClient.create(uri)
+    val database = mongoClient.getDatabase("sample_restaurants")
+    val collection = database.getCollection<Restaurant>("restaurants")
```

Comment:
suggestion: rename collection to "restaurants" so that it is clearer that we're inserting into the restaurants collection, especially since the user doesn't get to see the code declaring the collection/getting the collection

Reply:
if you do this, change the code that says "collection" to "restaurants"

Reply:
I would prefer not to do this, as different pages use different sample data (this repo uses restaurants and movies off the top of my head). I dont think that the code examples here are supposed to necessarily be runnable, moreso demonstrate/explain the API for performing certain tasks.

I think keeping the variable name as `collection` also makes it clear that you are performing the update operation on a `MongoCollection` instance aka at the collection lvl